### DISCUSSION TITLE
Customizable, short URLs for Stack Overflow profiles.

### DISCUSSION THREAD

**Discussion by choraria** [supabase/supabase#5689] [Show and tell]
Just opened-up [stk.so](https://stk.so/) to the world!

For anyone who's ever felt annoyed with the long profile URL on Stack Overflow, the tools allows you to create customizable, short URLs for them.

Made use of @supabase for the first time with this project and currently use it to host a database that maps the username/handle someone chooses to the Stack Overflow ID of the user.

Built it all using [Next.js](https://github.com/vercel/next.js/) + [Tailwind CSS](https://github.com/tailwindlabs/tailwindcss) and hosted on @vercel ⚡️

- [App site](https://stk.so/)
- [Shared on Twitter](https://twitter.com/choraria/status/1497892572865306628)
- [Shared on LinkedIn](https://www.linkedin.com/feed/update/urn:li:activity:6903653626502610944/)

https://user-images.githubusercontent.com/37455462/155880043-059ac4e3-23a7-4a59-ba19-f28aed4f882c.mp4

**Comment by niikhilpatel** [comment]
its good
